audit-ci.json

```diff
@@ -1,4 +1,4 @@
 {
     "high": true,
-    "retry-count": 20
+    "retry-count": 50
```

Comment:
We'll set this to something reasonable (or hopefully not need it) by the time the feature branch is merged.